Makefile

```diff
@@ -205,23 +204,27 @@ testconnectfastifyconformance: $(BUILD)/connect-fastify
 	npm run -w packages/connect-fastify conformance
 
 .PHONY: testwebconformance
-testwebconformance: testwebchromeconformance testwebfirefoxconformance testwebsafariconformance
+testwebconformance: testwebchromeconformance testwebfirefoxconformance testwebsafariconformance testwebnodeconformance
 
 .PHONY: testwebchromeconformance
 testwebchromeconformance: $(BUILD)/connect-web
-	npm run -w packages/connect-web conformance:client:chrome
+	npm run -w packages/connect-web conformance:client:chrome:promise
+	npm run -w packages/connect-web conformance:client:chrome:callback
 
 .PHONY: testwebfirefoxconformance
 testwebfirefoxconformance: $(BUILD)/connect-web
-	npm run -w packages/connect-web conformance:client:firefox
+	npm run -w packages/connect-web conformance:client:firefox:promise
+	npm run -w packages/connect-web conformance:client:firefox:callback
 
 .PHONY: testwebsafariconformance
 testwebsafariconformance: $(BUILD)/connect-web
-	npm run -w packages/connect-web conformance:client:safari
+	npm run -w packages/connect-web conformance:client:safari:promise
+	npm run -w packages/connect-web conformance:client:safari:callback
 
-.PHONY: testwebconformancelocal
```

Comment:
This didn't work as documented. Since the npm script runs two commands with `&&`, the `--browser` flag was only appended to the second command. The replacement is explained in packages/connect-conformance/README.md